src/domain/event/mutation.ts

```diff
@@ -82,38 +65,22 @@ export const MUTATION_EVENT = gql`
           id
           internalId
           name {
-            en
-            fi
-            sv
+            ...localisedFields
           }
           shortDescription {
-            en
-            fi
-            sv
+            ...localisedFields
           }
           description {
-            en
-            fi
-            sv
+            ...localisedFields
           }
           images {
-            id
-            internalId
-            license
-            name
-            url
-            cropping
-            photographerName
-            altText
+            ...imageFields
           }
           pEvent {
-            id
-            neededOccurrences
+            ...pEventFields
```

Comment:
Here we are adding some redundant fields that we don't actually need, which could have performance implications

Reply:
I reverted changes from pEvent -hunk